charts/vsphere-cpi/v1.0.0/values.yaml

```diff
@@ -3,7 +3,7 @@ vCenter:
   insecureflag: "1"
   port: "443"
   datacenters: ""
-  username: ""
-  password: ""
+  credentialsSecretName: ""
+  generateCredentialsSecret: false
```

Comment:
shouldn't the default be true here? This would deploy a broken chart by default since the credentialSecretName is not set and we are not creating a secret by default.